src/res/translation/wininstaller/sp.nsi

```diff
@@ -0,0 +1,37 @@
+; SPANISH translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_SPANISH} \
+    "Crear acceso directo en Escritorio"
+
+LangString INVALID_FOLDER_MSG ${LANG_SPANISH} \
+    "La carpeta de destino ya existe. Por favor escoja una carpeta de destino nueva."
+
+LangString RUNNING_APP_MSG ${LANG_SPANISH} \
+    "${APP_NAME} se está ejecutando. Por favor ciérralo y ejecuta la instalación de nuevo."
+
+LangString OLD_WRONG_VER_FOUND ${LANG_SPANISH} \
+    "Hemos detectado una versión antigua de ${APP_NAME} en tu carpeta de Archivos de Programa de 32 Bit. Se recomienda encarecidamente que la elimines antes de instalar una nueva versión de ${APP_NAME}. ¿Quieres eliminarla ahora?"
+
+LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_SPANISH} \
+    "Si continúas sin eliminarla, tu instalación podría fallar. ¿Estás segur@ de que no quieres eliminar la versión antigua?"
+
+LangString OLD_VER_REMOVE_FAILED ${LANG_SPANISH} \
+     "Lo sentimos, no ha sido posible desinstalar la versión antigua. Intentaremos instalar la nueva, pero también puedes pulsar en cancelar e intentar eliminar tú la versión antigua."
+
+LangString ASIO_DRIVER_HEADER ${LANG_SPANISH} \
+     "Driver ASIO"
+
+LangString ASIO_DRIVER_SUB ${LANG_SPANISH} \
+     "Para utilizar Jamulus, necesitas un driver ASIO"
+
+LangString ASIO_DRIVER_EXPLAIN ${LANG_SPANISH} \
+     "Jamulus necesita un driver ASIO para proporcionar una baja latencia. Más información:"
+
+LangString ASIO_DRIVER_MORE_INFO ${LANG_SPANISH} \
+     "Más información sobre ASIO en jamulus.io"
+
+LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_SPANISH} \
+     "https://jamulus.io/wiki/Installation-for-Windows#asio"
```

Comment:
I think, you can even link the spanish site here.

Reply:
Done.

Reply:
Great! Thanks. You can test the windows installer if you click on the checks page and download the artifact once the CI finished and do some tests.

Reply:
I don't have Windows but the installer works with WINE - looks ok.

Reply:
Ok. Great thank you.